lib/jellyfishService.js

```diff
@@ -74,6 +93,26 @@ var jellyfishService = function(envConfig, mongoClient, seagullClient, userApiCl
       if(this.theServer) {
         this.theServer.keepAliveTimeout = 151 * 1000;
         this.theServer.headersTimeout = 155 * 1000; // This should be bigger than `keepAliveTimeout + your server's expected response time` = 61 * 1000;
+
+        createTerminus(this.theServer, {
+          healthChecks: {
+            '/status': () => {
```

Comment:
If you define your healthcheck function as suggested above you don't need this entire code block - you can have just `'/status': healthCheck,`